oak_restricted_kernel_dice/src/lib.rs

```diff
@@ -23,10 +23,15 @@
 
 extern crate alloc;
 
+use alloc::vec;
+
```

Comment:
nit: extra newline? Or do you treat alloc differently?

Reply:
automatically formatted by [rustfmt](https://github.com/rust-lang/rustfmt). its ways are mysterious

Reply:
The new format used by rustfmt groups core/alloc/std together, all third-party ctates together and all local `crate::` imports together.